priv/prod.ssl_dist_opts.rel

```diff
@@ -1,15 +1,21 @@
 [
     {server, [
         {cacertfile, "/opt/braidnet/certs/braidcert.CA.pem"},
-        {certfile, "/opt/braidnet/certs/braidnet.pem"},
-        {keyfile, "/opt/braidnet/certs/braidnet.key"},
+        {certs_keys, [
+            #{certfile =>  "/opt/braidnet/certs/braidnet.pem",
+              keyfile => "/opt/braidnet/certs/braidnet.key"
+            }
+        ]},
         {verify, verify_peer},
         {secure_renegotiate, true}
```

Comment:
Shouldn't we add `{fail_if_no_peer_cert, true}` ?

Reply:
yes